RELEASING.md

```diff
@@ -35,5 +35,19 @@ Failure to do so will leave things in a broken state.
     git push upstream
     ```
 
+## Versioning:
+
+Once tag is created, it's time to use that tag for Runtime Versioning
```

Comment:
I'm not sure I followed the previous discussion well, but one subtle question:

Is this bumping the submitted version for the NEXT round of iterative development, or is this updating the existing version fields for the CURRENT release?  If the later, shouldn't that be done prior to creating the tag?

Reply:
I think @jsuereth is right, the version info should either be updated before creating the tag, or the tag should be updated after updating the version info.

Reply:
I have added the last step to move the tag to newly created commit-hash. Thanks for pointing out.